steps/api/epf/epf-api.ts

```diff
@@ -0,0 +1,20 @@
+import { type APIRequestContext, expect, request } from '@playwright/test'
+import { getToken } from '../auth/get-token.js'
+
+async function getContext(): Promise<APIRequestContext> {
+    const token = await getToken()
+    return request.newContext({
+        baseURL: process.env.EPF_API,
+        extraHTTPHeaders: {
+            Accept: 'application/json',
+            Authorization: `Bearer ${token}`,
+        },
+    })
+}
+
+export async function epfContext(crn: string, eventNumber: string): Promise<any> {
```

Comment:
I don't think it matters too much, but we're getting a warning here for using the `Any` type.  Is it worth creating an interface for the API response?

Reply:
ok done